lib/python/grassdb/checks.py

```diff
@@ -58,8 +58,8 @@ def is_location_valid(database, location):
     )
 
 
-def is_mapset_users(mapset_path):
-    """Check if the mapset belongs to the user"""
+def is_different_mapset_owner(mapset_path):
+    """Check if the mapset belongs to the different owner than the current user."""
```

Comment:
Why did you need to change `is_mapset_users()` into `is_different_mapset_owner()`? If it is just about clarity of the name then `is_current_user_mapset_owner()` might be better. If it is just about which case it reports as true, then mapset being owned by the current users is the primary or positive case, so I would have function for that and negate the result if need. (The point here is that this is a general functionality/API, not limited to data catalog.) If you really feel like function for the opposite case is needed, then you can wrap the existing one.

Besides the naming and what it is supposed to do, the current implementation is not correct: The GRASS_SKIP_MAPSET_OWNER_CHECK case does not correspond to `is_different_mapset_owner()`.

Reply:
I preserved the function is_mapset_users() but renamed it to is_current_user_mapset_owner(). Then I made the wrapping  function called is_different_mapset_owner().